README.md

```diff
@@ -43,6 +43,47 @@ Running Edge in Docker with our recommended setup:
 docker run -it -p 3063:3063 -e STRICT=true -e UPSTREAM_URL=<yourunleashinstance> unleashorg/unleash-edge:<mostrecentversion> edge
 ```
 
+## Edge behaviors
+
+As of version 19.2.0, Unleash Edge now supports two behaviors when running in edge mode: **strict** and **dynamic**. We recommend adopting the new **strict** behavior, while **dynamic** remains as a legacy option that will be deprecated and removed in a future release.
+
+For legacy reasons, **dynamic** behavior is still the default. However, a warning will be logged at startup to indicate its deprecation.
+
+Please note that these behaviors are mutually exclusive.
+
+### Strict behavior
+
+If started with the `--strict` flag or the `STRICT` environment variable, Edge now starts with strict behavior and must
+be
+given
+tokens at startup.
+Edge will lock down access for new tokens that have a wider or different access scope than the tokens it was started
```

Comment:
I think this is cool but I'd like to get a bit more specific in terms of what access means here. We do a nice job providing examples but I think we can do even better just by explaining what this is.

Let's lift this:

> Strict behavior still allows using tokens (both frontend and client) with narrower access than what the tokens it was started with has.

And combine it with this section and then build on that. Something like...?

Edge will refuses requests from SDKs that have a wider or different access scope than the initial tokens. Specifically, this means incoming requests must have a token that exactly matches the environment and is bound to a project or projects specified in that token.

Reply:
I think we can also tell a story on why this is valuable. Struggling to not go "full marketer" here but something like "this gives you more predictable control over which SDKs have access to Edge and prevents you from expanding the scope at runtime accidentally"